qa/crontab/teuthology-cronjobs

```diff
@@ -138,6 +141,12 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 ###### on smithi
 
 
-# 23 14 * * 6 CEPH_BRANCH=main; MACHINE_NAME=smithi; SUITE_NAME=upgrade:pacific-x; KERNEL=distro; /home/teuthology/bin/cron_wrapper /home/teuthology/bin/schedule_subset.sh 10 $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL -p 70 --force-priority
+23 14 * * 6 CEPH_BRANCH=reef; MACHINE_NAME=smithi; SUITE_NAME=upgrade:pacific-x; KERNEL=distro; /home/teuthology/bin/cron_wrapper /home/teuthology/bin/schedule_subset.sh 10 $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL -p 70 --force-priority
```

Comment:
This does not work
If you try scheduling it shows:
                                           
```
teuthology@teuthology:~$ CEPH_BRANCH=reef; MACHINE_NAME=smithi; SUITE_NAME=upgrade:pacific-x; KERNEL=distro; /home/teuthology/bin/schedule_subset.sh 10 $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL -p 70 --force-priority
-su: /home/teuthology/bin/schedule_subset.sh: No such file or directory
```

You can troubleshoot you or change to the old format, e.g. w/o schedule_subset.sh